contracts/validators/CredentialAtomicQueryValidator.sol

```diff
@@ -6,18 +6,32 @@ import {GenesisUtils} from "../lib/GenesisUtils.sol";
 import {ICircuitValidator} from "../interfaces/ICircuitValidator.sol";
 import {IVerifier} from "../interfaces/IVerifier.sol";
 import {IState} from "../interfaces/IState.sol";
+import {PoseidonFacade} from "../lib/Poseidon.sol";
 
 abstract contract CredentialAtomicQueryValidator is OwnableUpgradeable, ICircuitValidator {
+    struct CredentialAtomicQuery {
+        uint256 schema;
+        uint256 claimPathKey;
+        uint256 operator;
+        uint256 slotIndex;
+        uint256[] value;
+        uint256 queryHash;
+        uint256[] allowedIssuers;
+        string[] circuitIds;
+    }
+
     IVerifier public verifier;
```

Comment:
We have only one `verifier` state var here but potentially many verifier according to `CredentialAtomicQuery` struct. This looks like a wrong design. Need to discuss in offline.